### PR TITLE
tests: blacklist mtd using tests for esp32-wroom-32

### PR DESCRIPTION
**Pull request by kaspar030**
### Contribution description

These tests fail on CI. Blocklisting them for now so CI won't stumble all the time.

### Issues/PRs references

Corresponding issue: https://github.com/RIOT-OS/RIOT/issues/15123

### CODE REVIEW

**Review by benpicco — APPROVED** [review]
If nobody comes up with a better solution (those tests succeed locally) this is the easy way out :\

File: tests/pkg_littlefs/Makefile [hunk 1]
```diff
@@ -3,4 +3,6 @@ include ../Makefile.tests_common
 USEMODULE += littlefs
 USEMODULE += embunit
 
+TEST_ON_CI_BLACKLIST += esp32-wroom-32
```
Comment (benpicco) [inline tests/pkg_littlefs/Makefile:6]: Please add a comment as to why they are blacklisted (and that this should be undone ASAP)

Comment (kaspar030) [reply tests/pkg_littlefs/Makefile:7]: done: "# disabling due to unknown failure. See #15123."

**Comment by benpicco** [comment]
Turns out esp* did indeed break by #14754 - we should have trusted the CI on this one :disappointed:

**Pull request merged**